### PR TITLE
Gravity Form ajax spinner

### PR DESCRIPTION
**Pull request by anoblet**
#200

### CODE REVIEW

**Comment by kylebrodeur** [comment]
Task linked: [CU-232pth1 GForm AJAX spinner](https://app.clickup.com/t/232pth1)

File: packages/cxl-lumo-styles/scss/gravity-forms.scss [hunk 1]
```diff
@@ -18,6 +18,26 @@
...
 }
 
+/**
+ * Built-in spinner tweaks.
+ *
+ * @todo: remove inline styles from customizer,
+ * Spinner hotfix @kyle 2020.07.16.
+ * @see https://app.clickup.com/t/232pth1
+ * @since 2022.02.15
+ */
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/gravity-forms.scss:28]:
```suggestion
```

None of other styles has such extensive comments and I don't see the reason for this one being  special.

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

**Comment by pawelkmpt** [comment]
@anoblet just a small reminder about branch name conventions: https://app.clickup.com/2317190/v/dc/26pw6-8023/26pw6-7121

```diff
- feat/anoblet/spinner
+ anoblet/feat/spinner
```

File: packages/cxl-lumo-styles/scss/gravity-forms.scss (hunk 1 above)
Comment (pawelkmpt) [reply packages/cxl-lumo-styles/scss/gravity-forms.scss:28]: @anoblet when you'll have time for a tweak?

Comment (anoblet) [reply packages/cxl-lumo-styles/scss/gravity-forms.scss]: @pawelkmpt Updated

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**